### ISSUE TITLE
Rails 8 Support

### ISSUE THREAD

**Issue by ryanb** [salsify/goldiloader#162]
With [Rails 8.0 released](https://rubyonrails.org/2024/11/7/rails-8-no-paas-required) yesterday, any chance we can get a release of Goldiloader gem that doesn't have the `< 8` Rails version constraint?

For those wanting to upgrade now, we can point to master branch in the meantime.

```rb
gem "goldiloader", github: "salsify/goldiloader", branch: "master"
```

Thanks for your work on this awesome gem!

**Comment by jturkel** [comment]
Released in [5.4.0](https://github.com/salsify/goldiloader/releases/tag/v5.4.0).

**Comment by ryanb** [comment]
Wow that was fast, thanks @jturkel!
